sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceResolution.scala

```diff
@@ -85,6 +85,28 @@ case class DataSourceResolution(
           .asTableCatalog
       convertCTAS(catalog, identifier, create)
 
+    case DescribeColumnStatement(
+        AsTableIdentifier(tableName), colName, isExtended) =>
+      DescribeColumnCommand(tableName, colName, isExtended)
+
+    case DescribeColumnStatement(
+        CatalogObjectIdentifier(Some(catalog), ident), colName, isExtended) =>
+      throw new AnalysisException("Describing columns is not supported for v2 tables.")
+
+    case DescribeTableStatement(
+        AsTableIdentifier(tableName), partitionSpec, isExtended) =>
+      DescribeTableCommand(tableName, partitionSpec, isExtended)
+
+    case DescribeTableStatement(
+        CatalogObjectIdentifier(Some(catalog), ident), partitionSpec, isExtended) =>
+      if (partitionSpec.nonEmpty) {
+        throw new AnalysisException("DESCRIBE TABLE... PARTITION is not supported for v2 tables.")
+      }
+      DescribeTable(catalog.asTableCatalog, ident, isExtended)
+
+    case DropTableStatement(AsTableIdentifier(tableName), ifExists, purge) =>
```

Comment:
This was missing? Are there no tests for `DROP TABLE`?

Reply:
Oh this might have been a bad copy-paste artifact. Even if this is missing, it doesn't belong in this PR.